crates/attestation/src/crypto/crl.rs

```diff
@@ -272,6 +290,11 @@ mod tests {
                     critical: true,
                     extn_value: &bc,
                 },
+                x509::ext::Extension {
+                    extn_id: const_oid::db::rfc5912::ID_CE_CRL_DISTRIBUTION_POINTS,
+                    critical: true,
```

Comment:
Nitpick: CRL Distribution Point extensions should not be critical.

Reply:
Fixed

Reply:
Fixed, was in two places